### ISSUE TITLE
New Comment Animation after clicking on the reply button in comment section

### ISSUE THREAD

**Issue by 2underEST** [scbasis/scbasis-mini-program#2]

**Comment by Nekotizimo** [comment]
I agree that users should see the comment being replied to, but ideally, users should see both the post and the comment. But considering the space the keyboard takes up, we can't show both on screen at the same time. 

Maybe include in the header a "post" tab and a "comment" tab and the user can switch between them? idrk

The animation part (comment textarea pop up from the bottom) should be doable with css.

**Comment by 2underEST** [comment]
Cool, I think we can just the user scroll up and down if they wanna look at the post after clicking the reply button

**Comment by Nekotizimo** [comment]
Sure. So after they press reply they'll see the comment, the textarea, and their keyboard. Every other comment will be collapsed/hidden so that when they scroll up it's immediately the post?

Reddit closes the keyboard when you scroll up but I find this really annoying. If theres a way to keep the keyboard open when you scroll up that would be nice.

**Comment by crocodile27** [comment]
ok cool, who's doing this?

**Comment by crocodile27** [comment]
sry
